docs/UsersGuide/source/WE2Etests.rst

```diff
@@ -183,35 +184,70 @@ in the directory ``ufs-srweather-app/regional_workflow/tests/WE2E``.
 a CSV (Comma-Separated Value) file named ``WE2E_test_info.csv`` that contains information 
 on the full set of WE2E tests.  This file serves as a single location where relevant 
 information about the WE2E tests can be found. It can be imported into Google Sheets 
 using the "|" (pipe symbol) character as the custom field separator. The rows of the 
 file/sheet represent the full set of available tests (not just the ones to be run), 
 while the columns contain the following information (column titles are included in the CSV file):
 
-| Column 1:
-| The primary test name and (in parentheses) the category subdirectory it is
-  located in.
+| **Column 1**
+| The primary test name followed by the category subdirectory it is
+  located in (the latter in parentheses).
 
-| Column 2:
-| Any alternate names for the test (if any) followed by their category subdirectories
+| **Column 2**
+| Any alternate names for the test followed by their category subdirectories
   (in parentheses).
 
-| Column 3:
+| **Column 3**
 | The test description.
 
-| Column 4:
-| The number of times the forecast model will be run by the test.  This gives an idea
-  of how expensive the test is. It is calculated using quantities such as the number
-  of cycle dates (i.e. forecast model start dates) and the number of of ensemble members
-  (if running ensemble forecasts). The are in turn obtained directly or indirectly
-  from the quantities in Columns 5, 6, ....
+| **Column 4**
+| The relative cost of running the dynamics in the test.  This gives an 
+  idea of how expensive the test is relative to a reference test that runs 
+  a single 6-hour forecast on the ``RRFS_CONUS_25km`` predefined grid using 
+  its default time step.  To calculate the relative cost, the absolute cost 
+  ``abs_cost`` is first calculated as follows:
 
-| Columns 5,6,...:
-| The values of various experiment variables (if defined) in each test's configuration
-  file. Currently, the following experiment variables are included:
+.. code-block::
+
+     abs_cost = nx*ny*num_time_steps*num_fcsts
+
+| Here, ``nx`` and ``ny`` are the number of grid points in the horizontal 
+  (``x`` and ``y``) directions, ``num_time_steps`` is the number of time 
+  steps in one forecast, and ``num_fcsts`` is the number of forecasts the 
+  test runs (see Column 5 below).  [Note that this cost calculation does 
+  not (yet) differentiate between different physics suites.]  The relative 
+  cost ``rel_cost`` is then calculated using
+
+.. code-block::
+
+    rel_cost = abs_cost/abs_cost_ref
+
+| where ``abs_cost_ref`` is the cost of running the reference forecast 
+  described above, i.e. a single (``num_fcsts = 1``) 6-hour forecast on the 
+  ``RRFS_CONUS_25km grid`` (which currently has ``nx = 219``, ``ny = 131``, 
+  and a time step ``DT_ATMOS =  40 sec`` (so that ``num_time_steps = 6*3600/40 
+  = 540``), i.e.
+
+.. code-block::
+
+    abs_cost_ref = 219*131*540*1 = 15,492,060
+
```

Comment:
Is there some weird italics going on for abs_cost and abs_cost_ref?

Reply:
I'm not sure what's going on with the italics here in github, but it's just a text file and it looks fine after I use sphinx to generate the html.

Reply:
Thanks for confirming!